antora-playbook.yml

```diff
@@ -12,7 +12,7 @@ content:
     start_path: doc
   - url: https://github.com/clojure-emacs/cider-nrepl.git
     branches: master
-    tags: ['v0.24.0', 'v0.25.1', 'v0.26.0']
+    tags: ['v0.24.0', 'v0.25.1', 'v0.26.0', 'v0.37.0']
```

Comment:
That's not going to work, because you haven't be updating the antora version in `cider-nrepl` when cutting releases.